src/functionsDeployHelper.ts

```diff
@@ -161,10 +184,124 @@ export function getFunctionsInfo(parsedTriggers: CloudFunctionTrigger[], project
         "functions",
         trigger.name,
       ].join("/");
-      functionsInfo.push(triggerDeepCopy);
+      if (!_.get(regionMap, region)) {
+        regionMap[region] = [];
+      }
```

Comment:
```suggestion
      regionMap[region] = regionMap[region] || [];
```

You might be able to do `||=` but not sure if that's built-in to the version of Typescript we're using.

Reply:
Will do - I checked, and ||=, &&=, and ??= were added in 4.0 https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-0.html, but we're still on 3.9.5, so no luck there.